crates/holochain_p2p/src/types.rs

```diff
@@ -1,3 +1,7 @@
+/// Type that allows for mocking a network using
+/// [`MockHolochainP2pDnaT`]
+pub type GenericNetwork = Arc<dyn HolochainP2pDnaT>;
```

Comment:
I named this type spontaneously during our session. I'm happy enough with it, but open to suggestions =)